### PR TITLE
Updated README formatting & content

### PR DESCRIPTION
**Pull request by collingsr**
Cleaned up the formatting and content of the README page so it's easier to parse for potential new users. The focus was on highlighting features other mastodon apps don't have and all the customization options while keeping it succinct and clean. 

Still needs some screenshots inserted which are described in [ placeholder boxes ] - I was hoping Lucas could take them with the official moshidon account, if you drop them in a folder I can do any cropping/editing and putting the links into the actual readme before merging.

I also moved the list that used to be in the readme of detailed changes with the links to specific issues/pull requests into the FAQ in case you wanted to keep that part for your own reference. 

It's been a while since I've done a github PR so feedback / corrections are fine!

### CODE REVIEW

**Review by FineFindus — COMMENTED** [review]
Hey, first of all, thanks for the PR.

I added a few comments. Regarding `FAQ.md`, since most users are likely to switch from the official app to Moshidon, I don't think it makes sense to list features that are already available in the official app (especially since most of them are very basic). This is probably a lot to ask, but since the title says to update the content, the list is currently very outdated, maybe you could update it :)?

Also, if possible, please clean up the commit history (and ideally use [conventional commits](https://www.conventionalcommits.org/)).

File: FAQ.md [hunk 1]
```diff
@@ -7,3 +7,57 @@ A: There are many, but the most outstanding differences are: the ability to have
...
+
+* [Adding the ability to view other server's local timelines](https://github.com/LucasGGamerM/moshidon/tree/feature/local-timelines)
+* [Adding the ability to load followers and following from remote instance](https://github.com/LucasGGamerM/moshidon/tree/feature/remote-followers)
+* [Adding the ability to have filtered posts show with a warning](https://github.com/LucasGGamerM/moshidon/tree/feature/filters_again)
+* [Add “Unlisted” as a post visibility option](https://github.com/mastodon/mastodon-android/compare/master...sk22:megalodon:feature/enable-unlisted)
+  ([Pull request](https://github.com/mastodon/mastodon-android/pull/103))
+* Adding a useful private profile note box
+* Auto hiding the compose button on scroll
+* Adding the ability to remind yourself to add alt text to images
+* An indicator for if an image has alt text or not
```
[the remark this answers is not in the document]
Comment (LucasGGamerM) [reply FAQ.md:23]: Only the third one is on upstream, we have the others

[the remark this answers is not in the document]
Comment (FineFindus) [reply FAQ.md:23]: Yeah, I had some trouble with the selection UI :)

**Comment by collingsr** [comment]
Originally I had planned to work on the FAQ later in a separate PR but included it in this one because I copy pasted that list of features with links from the original readme into it. I wasn't sure if you actually wanted to keep it or not, because it's more detail than most users care about. Would it make sense to replace that list of features with a list of closed pull requests marked as "feat"? (I see the appeal of the conventional commits now). That would at least make sure the list is of all "moshidon" features and not from the base mastodon.

File: README.md [hunk 2]
```diff
@@ -1,185 +1,91 @@
-![MoshidonLogo](mastodon/src/main/res/mipmap-xhdpi/ic_launcher_round.png)
+# ![MoshidonLogo](mastodon/src/main/res/mipmap-xhdpi/ic_launcher_round.png) Moshidon, the material you mastodon client!
 
-# Moshidon, the material you mastodon client! 
 
-> A fork of [megalodon](https://github.com/sk22/megalodon) which is a fork of [official Mastodon Android app](https://github.com/mastodon/mastodon-android) adding important features that are missing in the official app and possibly won’t ever be implemented, such as the federated timeline, unlisted posting, bookmarks and an image description viewer.
+> A fast, highly customizable, up-to-date fork of [megalodon](https://github.com/sk22/megalodon) adding important features such as a fully federated timeline, unlisted posting, drafts, scheduled posts, bookmarks, and alt text warnings.
```
[the remark this answers is not in the document]
Comment (collingsr) [reply README.md:4]: There was a paragraph in the original readme about how the official mastodon app doesn't include the federated timeline, I didn't double check to see if that was still true.

**Comment by LucasGGamerM** [comment]
> Originally I had planned to work on the FAQ later in a separate PR but included it in this one because I copy pasted that list of features with links from the original readme into it. I wasn't sure if you actually wanted to keep it or not, because it's more detail than most users care about. Would it make sense to replace that list of features with a list of closed pull requests marked as "feat"? (I see the appeal of the conventional commits now). That would at least make sure the list is of all "moshidon" features and not from the base mastodon.

Well, the F.A.Q. should be kept for the essential things, like "What are the differences between megalodon and moshidon", and or other things users like to ask. 

Though, as you may have noticed, I have no idea how to write documentation, so I will leave it to you whether or not you'd wish to keep it like that or move the detailed feature set to a separate file.

**Comment by LucasGGamerM** [comment]
After a bit of thought, this should be fine to merge

**Comment by collingsr** [comment]
I'm sorry, I haven't abandoned this yet! I meant to get back to this sooner
but some other things came up, I even have some changes already made based
on the suggestions that I haven't pushed to github yet. I'll make another
PR when I've got enough changes to make it worthwhile. And thanks for the
shout out on masto! 😄

**Pull request merged**